DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -32,8 +32,7 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/DuckDuckGo/BrowserServicesKit",
       "state" : {
-        "revision" : "c6ce430371032930d770b0388cbe44a2d40ad729",
-        "version" : "184.0.0"
+        "revision" : "53e333ca0ffcb7da8947c439dfda23f3c0a16336"
```

Comment:
The core reason for this PR is to update BSK to match the latest changes from macOS.